inventory-manager.lic

```diff
@@ -223,6 +240,20 @@ class InventoryManager
         else
           container = item.to_s
         end
+      elsif /vault book/.match(command)
```

Comment:
Line 257 uses command == 'vault standard', the two new ones use /vault book/.match(command). I don't know this for sure, but .match does a regular expression comparison, which I suspect is more labor intensive than just doing a string compare. It's probably fairly insignificant, but I would think we're better off with consistency, in any event.

Reply:
One other thought, you should probably combine the vault book and storage book, since the code itself is the same.

Reply:
My initial thought was just being against"read my vault book" and "read my storage book" as the matches and typically just use regex.  Can change to whatever.

I was following the example of vault and family vault being unique methods when I made the updates.

Reply:
Oh, not the methods, I just mean these two elsif statements. But just my two cents!

```
      elsif /vault book/.match(command)
        # items in vault book output are prefixed by 6 spaces
        if line =~ /\s{6}(?:a|an|some) /
          item.concat(" (in container - #{container})")
        else
          container = item.to_s
        end
      elsif /storage book/.match(command)
        # items in a storage book are prefixed by 6 spaces
        if line =~ /\s{6}(?:a|an|some) /
          item.concat(" (in container - #{container})")
        else
          container = item.to_s
        end
```